ts/src/phemex.ts

```diff
@@ -2316,6 +2316,10 @@ export default class phemex extends Exchange {
             // 'posSide': Position direction - "Merged" for oneway mode , "Long" / "Short" for hedge mode
         };
         const clientOrderId = this.safeString2 (params, 'clOrdID', 'clientOrderId');
+        const stopLoss = this.safeValue (params, 'stopLoss');
```

Comment:
we're missing this order type in the js doc, we should add a reference to it
![image](https://github.com/ccxt/ccxt/assets/43336371/2ef9274f-87bc-4da3-ab2c-2abfb775840f)

Reply:
order type in the js doc?

Reply:
@AlgoQ probably something like ?
```
 * @param {object|undefined} params.takeProfit *swap only* *uses the Place Position TPSL* Object containing the price at which a take profit order is triggered at
 *  @param {float|undefined} params.takeProfit.triggerPrice take profit price
 ```